pool/pool.py

```diff
@@ -266,9 +271,14 @@ async def collect_pool_rewards_loop(self):
                 ph_to_coins: Dict[bytes32, List[CoinRecord]] = {}
                 not_buried_amounts = 0
                 for cr in coin_records:
+                    if not get_farmed_height(cr, self.constants.GENESIS_CHALLENGE):
```

Comment:
Couldn't we just check the `coinbase` flag of `CoinRecord` here? See https://github.com/Chia-Network/chia-blockchain/blob/main/chia/types/coin_record.py#L21

Reply:
Yeah that's easier. good point